src/dotnet/APIView/APIView/AttributeAPIV.cs

```diff
@@ -10,14 +10,14 @@ namespace APIView
     /// </summary>
     public class AttributeAPIV
     {
-        public string Type { get; set; }
-        public string[] ConstructorArgs { get; set; }
+        public TypeReference Type { get; set; }
+        public string[] ConstructorArgs { get; set; }  // TODO: Update to use TypeReference
```

Comment:
This shouldn't be a type reference. It represents constant values, not symbols.

Reply:
The constructor arguments?

Reply:
Hm, there two kinds of constructor arguments, some of them are constant values, some - named. I'm not sure exactly what is represented by the string right but TypeReference wouldn't be able to represent either named nor constant arguments.


`[Attribute(1, "Constant", null, NamedArgument="Value", AnotherNamedArgument = 1.5)]`